docs/4.4/api-reference.md

```diff
@@ -219,28 +547,43 @@ if err := client.DeleteToken(tokenString); err != nil {
 
 ## Cluster Labels
 
-The root cluster in a [trusted cluster](admin-guide.md#adding-nodes-to-the-cluster) can add/update cluster labels on leaf clusters.
+!!! Note
+    This feature will be release with Teleport 5.0.
+
+Cluster Labels can be used to differentiate between leaf clusters in a [Trusted Cluster](trustedclusters.md). These can be useful when defining [roles](enterprise/ssh-rbac.md) within a trusted cluster, where each cluster has its own requirements for access.
+
+For example, if each cluster corresponds to a different product that should only be accessed by its product team, you can give each cluster a label like `product: A`. Then create a role for each product which allows access to clusters with its respective product label.
+
+You may want to use the following RPC's to manage your cluster labels if:
```

Comment:
```suggestion
You may want to use the following RPCs to manage your cluster labels if:
```

Reply:
Should this be RPCs?

Reply:
Yes, RPC's with an apostrophe implies that something belongs to the RPCs ("an RPC's job is to allow remote procedure calls") whereas RPCs without an apostrophe is the plural of RPC.